pkg/specs/interface.go

```diff
@@ -348,6 +368,41 @@ func (r *Resolver) resolveUnforkRelease(
 	return release, nil
 }
 
+func (r *Resolver) maybeGetReleaseName(path string) (string, error) {
+	type k8sReleaseMetadata struct {
+		Metadata struct {
+			Labels struct {
+				Release string `yaml:"release"`
+			} `yaml:"labels"`
+		} `yaml:"metadata"`
+	}
```

Comment:
Any reason this type is defined in the function?

Reply:
it's only used here to take out the release name